ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -1583,7 +1583,71 @@ def arrays_for_pooling(
         padding = draw(st.sampled_from(["VALID", "SAME"]))
 
     strides = draw(st.tuples(st.integers(1, min(kernel))))
-    ret = [dtype, x, kernel, strides, padding]
+
```

Comment:
I'd also suggest we do this one in a seperate PR, to make things cleaner, this PR should only contain improving docstring.